app/tests/spec/models/account.js

```diff
@@ -325,21 +293,115 @@ function (chai, sinon, p, Constants, Assertion, ProfileClient,
         });
       });
 
+      it('can fetch an oauth access token', function () {
+        return account.createOAuthToken('scope')
+          .then(function (token) {
+            assert.equal(token.get('token'), accessToken);
+            assert.isTrue(assertion.generate.calledWith(SESSION_TOKEN));
+            assert.isTrue(oAuthClient.getToken.calledWith({
+              client_id: CLIENT_ID, //eslint-disable-line camelcase
+              scope: 'scope',
+              assertion: 'assertion'
+            }));
+          });
+      });
+
+      it('fails to if bad assertion', function () {
+        assertion.generate.restore();
+        sinon.stub(assertion, 'generate', function () {
+          return p.reject(AuthErrors.toError('UNAUTHORIZED'));
+        });
+        return account.createOAuthToken('scope')
+          .then(assert.fail,
+            function (err) {
+              assert.isTrue(AuthErrors.is(err, 'UNAUTHORIZED'));
+            });
+      });
+
+      it('fails to fetch when unauthorized', function () {
+        oAuthClient.getToken.restore();
+        sinon.stub(oAuthClient, 'getToken', function () {
+          return p.reject(AuthErrors.toError('UNAUTHORIZED'));
+        });
+        return account.createOAuthToken('scope')
+          .then(assert.fail,
+            function (err) {
+              assert.isTrue(AuthErrors.is(err, 'UNAUTHORIZED'));
+            });
+      });
+    });
+
+    it('isVerified returns true', function () {
```

Comment:
Aha, the tests for those other cases exist but are scattered about. I'll group 'em.